test/cloud_testing/platforms/centos7_x86_64_test.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+CVMFS_PLATFORM_NAME = "centos7_x86_64"
```

Comment:
Let's spell it `centos7-x86_64` with a dash as a separator between OS and architecture, so that we distinguish it from the underscore intrinsic to x86_64